chart/templates/frontend.yaml

```diff
@@ -63,9 +63,13 @@ spec:
 
           readinessProbe:
             httpGet:
-              path: /
-              port: 80
-             
+              path: /healthz
+              port: 8880
+            failureThreshold: 3
+            initialDelaySeconds: 10
```

Comment:
Isn't this too lengthy? I'm not certain what HTTP status code the Ingress controller will return for an access that occurs prior to its initial probing.

Reply:
Yeah, this is what the nginx ingress pod has, got it from there. There is some startup time to nginx, as it runs scripts, so probably should be >0?